src/docs/development/platform-integration/platform-views.md

```diff
@@ -26,7 +26,7 @@ Which one to use depends on the use case. Let's take a look:
   Certain platform interactions such as keyboard handling, and accessibility
   features might not work.
 
-* Hybrid composition requires Flutter 1.22. This mode appends the
+* Hybrid composition requires Flutter 1.22 (recommended version- 1.22.2). This mode appends the
```

Comment:
Could you link "version 1.22.2" to the GitHub wiki:  https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#1222--october-16-2020
Also, remove the hyphen "-".

So, the change should look like the following:

```
* Hybrid composition requires Flutter 1.22 ([version 1.22.2][] is recommended). 
  This mode appends the

[version 1.222]: {{site.github}}/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#1222--october-16-2020
```

Reply:
Ya Sure.

Reply:
@Sfshaza I have made changes as said. Have a look.